web/src/components/LandingPage/Footer.tsx

```diff
@@ -94,6 +97,6 @@ const Footer = () => {
       </div>
     </footer>
   );
-};
+});
 
-export default Footer;
+export default FooterComponent;
```

Comment:
This is mostly a matter of personal preference, but I think it is preferable not to use suffixes like "...Component", "...Class", "...Variable" when naming things.